packages/wonder-blocks-button/components/button-core.js

```diff
@@ -29,14 +29,18 @@ type Props = {|
     type?: "submit",
 |};
 
+type ContextTypes = {|
+    router: PropTypes.Requireable<any>,
```

Comment:
Technically, `PropTypes` is an untyped module, I believe so this is equivalent to just `any`, but it would be something like `PropTypes.Requireable<any>` if it were typed (I believe that's what TypeScript would use).

Reply:
I couldn't find anything about `Requireable` in https://github.com/facebook/prop-types.  Is this a made up type?

Reply:
It's what TypeScript says it should be. As in my comment, it really equates to `any` here.